Library/Homebrew/sandbox.rb

```diff
@@ -141,8 +141,15 @@ def path_filter(path, type)
   class SandboxProfile
     SEATBELT_ERB = <<-EOS.undent
       (version 1)
+      (deny network-inbound)
+      (deny network-outbound)
       (debug deny) ; log all denied operations to /var/log/system.log
       <%= rules.join("\n") %>
+      (allow network-outbound
+        (subpath "/tmp")
+        (subpath "/private")
+        (subpath "#{HOMEBREW_TEMP}")
```

Comment:
> Just wondering why you decided not to do this (or blocking any file reads) before?

Because I have not idea how to implement this without breaking it. It's much harder to create a whitelist on `read` operation than `write`.

> Any thoughts on this general approach and whether it will make sense? 

A good suggestion would be implementing methods such as `allow_read_*`, `allow_network_*` similar to `allow_write_*` instead of hard coding the rule.